### DISCUSSION TITLE
Remove the badges

### DISCUSSION THREAD

**Discussion by wwahammy** [community/community#18527] [Other Feature Feedback, Questions, & Ideas]
I think you folks mean well but turning Github into a social network style site is a terrible idea. Please stop going in this direction. I don't want all these badges on my profile. Let me turn them on or off.

And please stop with social networking stuff. We don't need it.

**Comment by iop3** [comment]
Go to https://github.com/settings/profile then scroll down. It gives you an option to turn achievements off.

**Comment by waterkip** [comment]
I actually liked the artic vault ones. But the PR shark is too gamey for my taste.

**Comment by arghanath007** [comment]
Yes I agree but it is also makes someone quickly make out if the person has some experience is contributing to open source projects

**Reply by SleepyMode** [reply]
> Yes I agree but it is also makes someone quickly make out if the person has some experience is contributing to open source projects

Or you could just look at their contributions.

**Reply by arghanath007** [reply]
Fair

**Comment by epistax1s** [comment]
Many people like it.

**Reply by rahji** [reply]
I teach art students who are new to coding and new to git/github. I think anything that makes this place feel more inviting to newcomers is a good thing - especially when it does not interfere, at all, with the experience of users who aren't excited about getting badges (ie: turn the feature off if you don't want to participate).

**Comment by xsqkv** [comment]
In the upper right corner, click on your avatar in the menu below, select settings, in the menu on the left, select a public profile and at the very bottom of the page there will be an option to show achievements in my profile, uncheck it and update preferences

**Comment by andomain** [comment]
I don't mind the concept but it would be nice to choose which badges appear on my profile, some are great but some look unprofessional. For example, I have one named 'Yolo' from merging a PR without review for a small personal project in which I am the only contributor. I would never do this for a proper project but having this on my profile advertises me as someone who does :(

**Reply by CapitainFlam** [reply]
It could indeed look unprofessional. I'm with on this 🤝 !

**Reply by rahji** [reply]
Yes, turn the feature off if you're worried about how professional it looks. Many students are up for having some fun with source control and will leave the feature on.

**Comment by tvlooy** [comment]
Please at least make it an option to completely disable everything that has to do with badges in github. I am here for git and code, not for getting badges. I predict that if you don't, there will be a browser plugin soon to do it.

**Reply by waterkip** [reply]
See https://github.com/orgs/community/discussions/18527#discussioncomment-2943877 on how to do just that.

Or just directly go to https://github.com/settings/profile and change it, it is located at the bottom of the page.

**Reply by tvlooy** [reply]
I don't care it is on my profile. I can ignore it there. I just logged in and got popups about badges I achieved. I do not want that.

**Comment by OOCAZ** [comment]
I don't know that there is a simple answer for this question. Some people like them, others don't. I think the option to mute or hide achievements and achievement notifications we can meet a happy middle ground. Sure they "game-ify" PR's to an extent. But the people who just want the badge will get it and move on with their life. No different than any one who just wanted to say they contributed to an open source project just to say they did. I think GitHub is very open for feedback on the different achievements and a few revisions to the achievements could help those looking to "game-ify" the experience a bigger contributor. But in the end it's just a cool thing to keep some people engaged.

**Comment by jfullstackdev** [comment]
latest update, you can now pick what you want to hide, unlike in the past, it was all or nothing

go to Achievements Tab, click the `Achievements` word, it is a link

![image](https://user-images.githubusercontent.com/47092464/204711684-e7c5ebb6-57f9-4fbf-b8fe-96c2182f41bb.png)

or this URL, it's same thing 

https://github.com/[username]?tab=achievements

you should be seeing like this 

![image](https://user-images.githubusercontent.com/47092464/204711801-dde96c4d-5133-4201-ad8a-9b23b55ed5d2.png)

then, click any badge, hover left then you should be seeing like this 

![image](https://user-images.githubusercontent.com/47092464/204711940-2c5e8b4d-7580-4145-afc0-268f1739d8de.png)

then boom ... you now have the option to hide those badges that you want to hide

**Reply by andomain** [reply]
Thats perfect, exactly what I was thinking

**Comment by FedericoCeratto** [comment]
At least the badges should be opt-in rather than opt-out.

**Reply by tvlooy** [reply]
If it was opt-in, nobody would activate it. That's why. Github should be smarter than this and just drop this entire unnecessary product of featuritis. It brings zero value and only caused distraction.

**Comment by hughesjs** [comment]
My dude... You can turn it off... Some of us like it

**Comment by DSDmark** [comment]
I agree. This discussion make sense. There are a lot of badges which are not useful, like **YOLO** and **Quick draw**. And **Pull Shark** also what its make sense if I create pull request in own branch and marge it. And also it's too easy to achieve these badges.

In my opinion, it would be fun if GitHub set a timer on these badges, like every month's last. All badges should disappear. If you are not work for it.
